gunicorn.conf.py

```diff
@@ -28,7 +28,10 @@ def start_websockify(websockify_path, target_file):
             ],
             stdout=subprocess.PIPE,
         )
+    else:
+        print("Websockify started.")
```

Comment:
See above

Reply:
I also want to keep this. There's no other indication that this actually ran.